### ISSUE TITLE
Add option for specifying extra content in the PR description

### ISSUE THREAD

**Issue by lcawl** [sqren/backport#96]
There are some comments that we add to the PR description and they affect the types of checks that are run immediately on the PR. This tool is so fast that I can't get into the backport PRs and add those strings (like "skip ci") before the (unnecessary) tests start running.

**Comment by sqren** [comment]
Possible solution:
Add a field to the config.json where users can specify text they want to add to the PR description:

```
{
  "accessToken": "...",
  "username": "sqren",
  "prDescription": "skip-ci"
}
```

**Comment by sqren** [comment]
@lcawl This has been released in version 4.5.2. You can install it with:
```
yarn global add backport
```

or 

```
npm install -g backport
```

Then you can supply the description like:
```
backport --prDescription="skip-ci"
```

Or add it to your config file (in ~/.backport/config.json):
```json
{
    username: "...",
    accessToken: "...",
    prDescription: "skip-ci"
}
```
